packages/authentication/types/allAuth.ts

```
@@ -0,0 +1,713 @@
+// import { MfaMethod } from './mfa'
+import { AxiosResponse } from 'axios'
+
+// Settings
+export const Client = Object.freeze({
+  APP: 'app',
+  BROWSER: 'browser',
+})
+
+// Config Types
+
+export interface ConfigRequest {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)